packages/react-ui/src/widgets/legend/LegendWrapper.js

```diff
@@ -52,6 +70,10 @@ export default function LegendWrapper({
     if (onChangeVisibility) onChangeVisibility({ id, visible: !visible });
   };
 
+  const handleToggleLayerOptions = () => {
```

Comment:
@Clebal Is this required to this PR?

Reply:
Yes, when layer options is expanded, the opacity control is visible.